spec/factories/provider.rb

```diff
@@ -1,7 +1,7 @@
 FactoryBot.define do
   factory :provider do
     code { Faker::Alphanumeric.unique.alphanumeric(number: 3).upcase }
-    name { Faker::University.name }
+    name { "#{Faker::University.name}-#{rand(9999)}" }
```

Comment:
```suggestion
    sequence(:name) { |n| "#{Faker::University.name}-#{n}"  }
```
How do you feel about using a [sequence](https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#inline-sequences) rather than random numbers?

Reply:
I may have misunderstood how sequence works -- i thought it was only used for lists, so that if you are just doing create(:provider) and then a few lines later create(:provider), n == 1 in both cases. Which would help, but not as much as the random number -- i initially just experimented with rand(10) and got fewer duplicates, but still a notable number.